Sources/CoffeeBot/CoffeeManager/CoffeeManager.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  CoffeeManager.swift
+//  SwiftBot
+//
+
+import Foundation
+import BotsKit
+import LoggerAPI
+
+internal protocol CoffeeManagerProtocol {
+    func process(activity:Activity)
+}
+
+class CoffeeManager {
+    
+    //MARK: Properties
+    fileprivate var session : Session? = nil
```

Comment:
I think there could be only one session at a time. Because if a second session is started before the first one is ended how will we differentiate between which coffee request came to which session?